docs/docs-content/enterprise-version/system-management/account-management/email.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_label: "Email Address"
```

Comment:
Since this page is a how-to guide on changing the email address, maybe the title should be "Change Email Address?"